### ISSUE TITLE
White screen and `Not responding` window at very start

### ISSUE THREAD

**Issue by F8ER** [microsoft/vscode#99524]
The VSCode just does not start anymore.
First of all, it worked well, but suddenly it froze and showed a message `Not responding`.
Then, I killed all VSCode processes and tried to run it. In the result it shows only white screen a nothing more. Even no window bar at the top.

This is how it looks like:
![](https://i.imgur.com/WeG5ggw.png)

- VSCode Version:
- OS Version: Win 10 64 bit 1909 18363.836

Steps to Reproduce:

1. Just launch the VSCode

Does this issue occur when all extensions are disabled?: Yes

**Comment by vv7v7** [comment]
So, after a bunch of time wasted to try run VSCode without reboot, I've just rebooted PC and VSCode has launched.

Although, this issue might be related to extensions' update process. After VSCode launch, I've noticed that `junstyle.php-cs-fixer` extension has been updated from `0.1.153` to `0.1.154`. Also, it's worth to notice that I had a setting inside `settings.json` of VSCode such like:
```
"php-cs-fixer.config": ".php_cs;C:\\Users\\P1\\.vscode\\extensions\\junstyle.php-cs-fixer-0.1.153\\php_cs.dist"
```
Thus, after an update, something might have not find the path and might have cause the lag even running idle(without any manipulations) for about 10 minutes or so.

**Comment by deepak1556** [comment]
Since this is related to extension and update process, looping in @sandy081 .If the issue traces back to the runtime, I can look into it.

**Comment by sandy081** [comment]
Can you please try to reproduce this by disabling this extension?

**Comment by lockwoodnd** [comment]
Fajdsifjq903

**Comment by lockwoodnd** [comment]
I see peoplen want more csam messages because I yet again received a email from this list. Very well then.

 

Faslkdfjhalksfdjop0iq34098q3uy

**Comment by lockwoodnd** [comment]
Asl;.ka;lsefj0opiqu324[90tru0289-=

**Comment by lockwoodnd** [comment]
I hope people like my posts. Hahahahahahaa

**Comment by F8ER** [comment]
> Can you please try to reproduce this by disabling this extension?

Sorry, but, reproduce what? I mean, it should update an extension, but how to install an old version of such?

**Comment by sandy081** [comment]
Can you please confirm if this related to the extension update process by doing following

- Open extensions folder using the command `Open Extensions Folder`
- Remove the extension 
- Install the extension again

Let me know if you can repro it?

**Comment by F8ER** [comment]
> Can you please confirm if this related to the extension update process by doing following
> 
> * Open extensions folder using the command `Open Extensions Folder`
> * Remove the extension
> * Install the extension again
> 
> Let me know if you can repro it?

No, the issue does not appear with this manipulations.

**Comment by F8ER** [comment]
@sandy081 And here we go again. It froze and now it does not even open. I see only white screen and the message:
![](https://i.imgur.com/hwUGUdg.png)
Also:
![](https://i.imgur.com/ip9K3s9.png)
This is when it froze:
![](https://i.imgur.com/LVQZbXa.png)

**Comment by F8ER** [comment]
This is portable x64 Windows `.zip` version. I even redownloaded it, but the issue persists. How to use it now?!

**Comment by F8ER** [comment]
The ways I have tried:
1. Kill all instances of VSCode and run again.
2. Redownload same latest version and run
3. Start with `--disable-extensions`
Nothing helped.

It started only after PC reboot, but with empty workspace while before this freeze it opened it automatically after each start!
Thus, it forced me to reopen the workspace manually.
I noticed that the above extension has been updated to `v0.1.155`, but I had not noticed was it updated before the freeze.
